blog/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness.md

```diff
@@ -0,0 +1,691 @@
+---
+title: Create a Lottery dApp using Celo Composer and Redstone's Randomness
+description: Solidity smart contracts can use randomness to ensure that lottery outcomes are fair and unbiased, hence the use of Redstone randomness, which allows for transparency in the implementation.
+authors:
+  - name: Justin Obi
+    title: Software Engineer / Technical Writer
+    url: https://github.com/cjustinobi
+    image_url: https://github.com/cjustinobi.png
+tags: [celosage, intermediate, react, hardhat, oracle]
```

Comment:
can you add all the tags you have in your user.tsx file here too, it has to be the same thing